app/commands/community_video/search.rb

```diff
@@ -22,8 +22,8 @@ def filter_criteria!
     return if criteria.blank?
 
     criteria.strip.split(" ").each do |crit|
-      @videos = @videos.joins(:author).where("community_videos.title LIKE ?", "%#{crit}%").
-        or(@videos.joins(:author).where("users.handle LIKE ?", "%#{crit}%"))
+      @videos = @videos.left_joins(:author).where("community_videos.title LIKE ?", "%#{crit}%").
+        or(@videos.left_joins(:author).where("users.handle LIKE ?", "%#{crit}%"))
```

Comment:
As author is nullable, this needs to be a `left_join`, else records without authors will never match (as `join` is inner join by default)